lib/repository/genre_repo.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+import '../constants/constants.dart';
+import './interface/genre_interface.dart';
+
+class GenreRepository extends GenreInterface {
+  late Map<int, String> genresById = {};
+
+  static Future<Map<int, String>> parseJsonGenres() async {
```

Comment:
This can be non-static, and even private

Reply:
Done